cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -209,7 +209,7 @@ visualization_msgs::MarkerArray MapBuilderBridge::GetTrajectoryNodesList() {
     marker.header.frame_id = node_options_.map_frame;
     marker.color.b = 1.0;
     marker.color.a = 1.0;
```

Comment:
By the way, what do you think about different colours for different trajectories? Maybe a TODO could be dropped here?